python/mlc_chat/compiler/extern/fastertransformer.py

```diff
@@ -0,0 +1,173 @@
+"""FasterTransformer library."""
+import dataclasses
+import operator
+from functools import reduce
+from typing import List, Optional
+
+from tvm.relax.frontend import nn
+
+SRC = """
+#include <cutlass_kernels/fpA_intB_gemm.h>
```

Comment:
we should probably link this cuda code into TVM runtime instead of linking to MLC's compilation artifact